backend/community/community/settings.py

```diff
@@ -0,0 +1,146 @@
+"""
+Django settings for community project.
+
+Generated by 'django-admin startproject' using Django 3.1.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
```

Comment:
It is not "explicitly said" anywhere but what I think we should do is extract settings hardcoded in this file (e.g. `SECRET_KEY`, `CELERY_URL`, `EMAIL_HOST`, etc) to e.g. separate (git-ignored) files (e.g. json) with configuration specific for develop, and production/

Reply:
I don't think it's ever good to expose this type of information on a public repo.

Reply:
That's right, it will be extracted to a separate env file as included in the PR description.

Until we have a deployment script / startup script having the values hardcoded makes testing easier.